src/website/src/app/documentation/demos/progress-bars/progress-bar-component.demo.html

```diff
@@ -32,6 +32,22 @@ <h4>API</h4>
     </tbody>
   </table>
 
+  <div class="alert alert-warning">
+    <div class="alert-items">
+      <div class="alert-item static">
+        <div class="alert-icon-wrapper">
+          <clr-icon class="alert-icon" shape="exclamation-triangle"></clr-icon>
+        </div>
+        <span class="alert-text">
+          As from V3 we no longer provide <code class="clr-code">aria-live</code> input.
```

Comment:
```Since v3, we no longer handle setting `aria-live` and announcing the message for you by default. Based on the application use case, you can use the new `ClrAriaLiveService` to make announcements when they make sense for a user to hear about updates or loading status changes.```